src/components/modals/TradePreviewModalGP.vue

```diff
@@ -18,6 +22,10 @@
         </div>
       </div>
       <div>
+        <div class="mt-6 mb-3 text-sm">
+          Fee: {{ fNum(formattedFeeAmount, 'token') }}
```

Comment:
The trade preview will contain the fee info, so I've added it here - but it isn't the final design.